### PR TITLE
update fairing to kubeflow-fairing

### PR DESCRIPTION
**Pull request by jinchihe**
Since the fairing has been a subpackage of kubeflow, renamed to kubeflow-fairing, see more in the PR https://github.com/kubeflow/fairing/pull/359 and ticket https://github.com/kubeflow/fairing/issues/234. And kubeflow-fairing 0.6.0 version has been released.

The PR is to update fairing to kubeflow-fairing, thanks.

Fixes: #1146 

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1155)

### CODE REVIEW

**Comment by jinchihe** [comment]
Reviewed the `preview`,  seems that's good to me.

**Comment by joeliedtke** [comment]
Staged pages for review:
https://deploy-preview-1155--competent-brattain-de2d6d.netlify.com/docs/fairing/
https://deploy-preview-1155--competent-brattain-de2d6d.netlify.com/docs/fairing/gcp/
https://deploy-preview-1155--competent-brattain-de2d6d.netlify.com/docs/fairing/install-fairing/

File: content/docs/fairing/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "Fairing"
+title = "Kubeflow Fairing"
```
Comment (joeliedtke) [inline content/docs/fairing/_index.md]: Please revert this change to keep "Fairing" as the page title.

In the Kubeflow docs, we leave Kubeflow out of the title for components/features such as Pipelines and Fairing to keep the page navigation concise and easy to use.

Comment (jinchihe) [reply content/docs/fairing/_index.md]: Rollbacked, thanks @joeliedtke

**Comment by jinchihe** [comment]
@joeliedtke Addressed your comments, and reviewed the `preview`, that looks good. Thanks.

**Comment by joeliedtke** [comment]
Thanks!

/lgtm
/approve
